connector_woocommerce/model/woo_binding/common.py

```diff
@@ -51,10 +51,8 @@ class WooBinding(models.AbstractModel):
 
     @job(default_channel='root.woo')
     @api.model
-    def import_batch(self, backend, filters=None):
+    def import_batch(self, backend, filters=dict()):
```

Comment:
any logic behind replacing None by dict()?

Reply:
Yes, Before when the filters is none, it is checked with if condition and set as empty dictionary. As of now if condition is removed, so made the filters as empty dictionary by default.
Before,
def import_batch(self, backend, filters=None):
        """ Prepare the import of records modified on Woo """
        if filters is None:
            filters = {}
After,
def import_batch(self, backend, filters=dict()):
        """ Prepare the import of records modified on Woo """